test/Integration/test-powershell/AddProductIncorrectCasing/run.ps1

```diff
@@ -0,0 +1,18 @@
+using namespace System.Net
```

Comment:
I suppose these should also have a copyright header. @VasuBhog Can you make an issue to follow up adding that for the powershell files? (ideally with some sort of linting to ensure it's not missed in the future...not sure what exists out there for that)

Reply:
Created a follow up issue here: https://github.com/Azure/azure-functions-sql-extension/issues/610